redirect.go

```diff
@@ -92,13 +92,13 @@ func ReleaseRedirect(r *Redirect) {
 }
 
 func (r *Redirect) release() {
-	r.status = 302
+	r.status = StatusSeeOther
```

Comment:
@andradei This makes me wonder if we forgot to call release() somewhere before your changes, and it got exposed now

Reply:
I think I found why

Reply:
A missing ReleaseCtx() was the problem. Actually all the tests needed it